testing/py-ostinato/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Maintainer: Corentin Henry <corentinhenry@gmail.com>
+pkgname=py-ostinato
+_pkgname=python-ostinato
+pkgver=0.7.1
+pkgrel=0
+pkgdesc="A python agent for the ostinato controller"
+url="http://ostinato.org/"
+arch="noarch"
+license="GPLv3"
+depends="python"
+makedepends="python-dev py-setuptools"
+source="http://pypi.python.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
+
+_builddir="$srcdir"/$_pkgname-$pkgver
```

Comment:
Please rename `_builddir` to `builddir` (this has been changed in abuild tool recently).